.chglog/config.yml

```diff
@@ -18,6 +18,7 @@ options:
       pat: 📖 Pattern Language
       chore: 🧹 Chore
       fix: 🐞 Fix
+      kaizen: 👩‍🎨 Improve
```

Comment:
Maybe we should introduce [commitlint](https://commitlint.js.org/) as CI action to make sure we are following this guideline.

Reply:
@embano1 what do you think?

Reply:
Great idea!